src/panel/context/Request.tsx

```diff
@@ -33,7 +33,9 @@ export const RequestProvider: FC = ({ children }) => {
     response?: Record<string, unknown>;
     error?: Record<string, unknown>;
   }>({ fetching: false, response: undefined, error: undefined });
-  const [query, setQuery] = useState<string>();
+  const [query, setQuery] = useState<string>(
+    localStorage.getItem("urql-last-request") || ""
+  );
```

Comment:
There's two states for the query attribute:
 - `undefined` - no user input yet
 - `string` - user input

We need to default to undefined for [this functionality](https://github.com/FormidableLabs/urql-devtools/blob/master/src/panel/pages/request/components/Query.tsx#L62).

```suggestion
  const [query, setQuery] = useState<string | undefined>(
    localStorage.getItem("urql-last-request") || undefined
  );
```

Not sure why TS never complained about the incorrect type definition 🤷